packages/edit-site/src/components/site-icon/index.js

```diff
@@ -36,7 +36,7 @@ function SiteIcon( { className } ) {
 	) : (
 		<Icon
 			className="edit-site-site-icon__icon"
-			size="48px"
+			size="60px"
```

Comment:
It might be more readable to define this value in an SCSS file (`packages/edit-site/src/components/site-icon/style.scss`).

Perhaps the value can be expressed in `$grid-unit-70 + $grid-unit-05` ?

Reply:
I think size is fine here in the `Icon` but it shouldn't be a px value. Fixed in https://github.com/WordPress/gutenberg/pull/58472/commits/839d198b4a220de4b01a8798e81622d80a9735ba.